plugin.py

```diff
@@ -1,6 +1,9 @@
-from LSP.plugin import AbstractPlugin
+from LSP.plugin.core.typing import Optional
+from LSP.plugin import AbstractPlugin, Request
 from LSP.plugin import register_plugin
 from LSP.plugin import unregister_plugin
+from LSP.plugin.core.registry import LspTextCommand
+from LSP.plugin.core.views import text_document_position_params
```

Comment:
Just a style nit but we typically:
 - sort the imports
 - have one import per line (apart from `typing` stuff). Kinda arbitrary choice but it became the convention. :)